### DISCUSSION TITLE
Conversion of LaTeX sections to Markdown not what I expect

### DISCUSSION THREAD

**Discussion by dmca-glasgow** [jgm/pandoc#9479] [Q&A]
Hi there,

Given the following LaTeX:

```latex
\documentclass{article}

\begin{document}

\begin{definition}
Test 1
\end{definition}

\begin{definition} \label{CMD1.1}
Test 2
\end{definition}

\end{document}
```

And running the command `pandoc --output test.md test.tex`

The resulting Markdown is:

```md
::: definition
Test 1
:::

::: definition
[]{#CMD1.1 label="CMD1.1"} Test 2
:::
```

This doesn't work with `pdflatex` though, resulting in:

```bash
! LaTeX Error: Environment definition undefined.
```

When I fix this by adding a `\newtheorem` line:

```latex
\documentclass{article}
\newtheorem{definition}{Definition}

\begin{document}

\begin{definition}
Test 1
\end{definition}

\begin{definition} \label{CMD1.1}
Test 2
\end{definition}

\end{document}
```

The resulting Markdown "definition" directive with the custom label has lost it's name and gained a class:

```md
::: definition
**Definition 1**. *Test 1*
:::

::: {#CMD1.1 .definition}
**Definition 2**. *Test 2*
:::
```

Is this a bug or intentional?  If intentional, why?  

I'd just like to know before I write functionality to handle it further down the chain.  

Thanks!

**Comment by jgm** [comment]
First, note that there is no semantic difference in pandoc between
```
::: definition
hi
:::
```
and
```
::: {.definition}
hi
:::
```
The first is just syntactic sugar for the second.  So all that is different here is that an identifier has been added.

**Reply by dmca-glasgow** [reply]
Ok, thanks for the clarification, I can handle that in my Markdown to HTML processor.

Just to note in case you are concerned about compatibility with the [unified](https://github.com/unifiedjs/unified)/[remark](https://github.com/remarkjs/remark) ecosystem, the official [remark-directive](https://github.com/remarkjs/remark-directive) plugin doesn't accept the latter syntax:

```md
::: definition
hi 1
:::

::: {.definition}
hi 2
:::
```

Resulting AST:

```json
{
  "type": "root",
  "children": [
    {
      "type": "containerDirective",
      "name": "definition",
      "attributes": {},
      "children": [
        {
          "type": "paragraph",
          "children": [
            {
              "type": "text",
              "value": "hi 1"
            }
          ]
        }
      ]
    },
    {
      "type": "paragraph",
      "children": [
        {
          "type": "text",
          "value": ":::{.definition}\nhi 2\n:::"
        }
      ]
    }
  ]
}
```
